### PR TITLE
Update BUILD.md to link to the wiki.

### PR DESCRIPTION
**Pull request by ZekeZDev**

### CODE REVIEW

**Comment by glowiak** [comment]
But why?

**Review by d-513 — APPROVED** [review]

**Review by Scrumplex — APPROVED** [review]

**Comment by d-513** [comment]
> But why?

All documentation is now maintained on wiki, for ease of use and contributing.

**Review by glowiak — APPROVED** [review]
ok

File: BUILD.md [hunk 1]
```diff
@@ -1,341 +1,5 @@
...
-```sh
-# configure everything
-cmake -S . -B build \
-   -DCMAKE_BUILD_TYPE=Release \
-   -DCMAKE_INSTALL_PREFIX="/usr/local" \ # /usr/local is default in OpenBSD and FreeBSD
-   -DLauncher_PORTABLE=OFF -DCMAKE_PREFIX_PATH=/usr/local/lib/qt5/cmake # use linux layout and point to qt5 libs
-cd build
-make -j$(nproc) install # Optionally specify DESTDIR for packages (i.e. DESTDIR=${pkgdir})
-```
+If you would like to contribute or fix an issue with the Build instructions your can do so [here](https://github.com/PolyMC/polymc.github.io/blob/master/src/wiki/development/build-instructions.md)
```
Comment (txtsd) [inline BUILD.md]: s/your/you/

**Review by txtsd — CHANGES_REQUESTED** [review]

**Review by txtsd — APPROVED** [review]

**Pull request merged**